src/main/java/javaserver/Routes.java

```diff
@@ -1,38 +1,28 @@
 package javaserver;
 
-import java.io.File;
 import java.util.HashMap;
-
 import javaserver.ResponseBuilders.*;
 
 public class Routes {
 
-	public static HashMap<String, ResponseBuilder> routeResponders = new HashMap<String, ResponseBuilder>();
+	private static HashMap<String, Responder> routeResponders = new HashMap<String, Responder>();
 	static {
-		routeResponders.put("/", new DirectoryResponseBuilder());
-		routeResponders.put("/parameters", new ParameterResponseBuilder());
-		routeResponders.put("/method_options", new OptionResponseBuilder());
-		routeResponders.put("/form", new FormResponseBuilder());
-		routeResponders.put("/redirect", new RedirectResponseBuilder());
-		for (String file : getDirectoryListing("public")) {
-			routeResponders.put("/" + file, new FileResponseBuilder());
+		routeResponders.put("/", new DirectoryResponder(new String[] {"GET"}, App.publicDirectory));
```

Comment:
It's a little strange that you're accessing an object through App from a property. Might be better to expose a method to retrieve this.